app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is defined as a class variable, which is fine for storing all instances. However, ensure that this approach aligns with your application's requirements, as it will persist data across all instances and may lead to unintended data retention.